src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] Please convert the App component into a class component.

Reply:
App component has been converted into a class